trade-finance-manager-api/src/graphql/helpers/amendment.helpers.api-test.js

```diff
@@ -0,0 +1,292 @@
+
+  it('should return the same number since in GBP', () => {
+    amendment.value = 25000;
+
+    const result = amendmentHelpers.calculateNewFacilityValue(null, amendment);
+
+    const expected = amendment.value;
+
+    expect(result).toEqual(expected);
+  });
+
+  it('should return a number if different currency', () => {
+    amendment.value = 25000;
+    amendment.currency = 'JPY';
+    const exchangeRate = 7.1;
+
+    const result = amendmentHelpers.calculateNewFacilityValue(exchangeRate, amendment);
+
+    const expected = amendment.value * exchangeRate;
+
+    expect(result).toEqual(expected);
+  });
+
+  it('should return null if no facility value or currency in amendment', () => {
+    amendment.value = null;
+    amendment.currency = null;
+    const exchangeRate = 7.1;
+
+    const result = amendmentHelpers.calculateNewFacilityValue(exchangeRate, amendment);
+
+    const expected = null;
+
+    expect(result).toEqual(expected);
+  });
+
+  it('should return null if no exchange rate if currency not GBP', () => {
+    amendment.value = 25000;
+    amendment.currency = 'JPY';
+    const exchangeRate = null;
+
+    const result = amendmentHelpers.calculateNewFacilityValue(exchangeRate, amendment);
+
+    const expected = null;
+
+    expect(result).toEqual(expected);
+  });
+});
+
+describe('calculateUkefExposure()', () => {
+  it('should return the cover percentage without rounding when not needed', () => {
+    const facilityValueInGBP = '5000';
+    const coverPercentage = 80;
+    const result = amendmentHelpers.calculateUkefExposure(facilityValueInGBP, coverPercentage);
+
+    const expected = 5000 * (80 / 100);
+
+    expect(result).toEqual(expected);
+  });
+
+  it('should return a rounded number when more than 2 d.p', () => {
+    const facilityValueInGBP = '5165.2';
+    const coverPercentage = 33;
+
+    const result = amendmentHelpers.calculateUkefExposure(facilityValueInGBP, coverPercentage);
+
+    const expected = 1704.52;
+
+    expect(result).toEqual(expected);
+  });
+
+  it('should return null if no facility value', () => {
+    const facilityValueInGBP = null;
+    const coverPercentage = 33;
+
+    const result = amendmentHelpers.calculateUkefExposure(facilityValueInGBP, coverPercentage);
+
+    const expected = null;
+
+    expect(result).toEqual(expected);
+  });
+
+  it('should return null if no cover percentage', () => {
+    const facilityValueInGBP = '5165.2';
+    const coverPercentage = null;
+
+    const result = amendmentHelpers.calculateUkefExposure(facilityValueInGBP, coverPercentage);
+
+    const expected = null;
+
+    expect(result).toEqual(expected);
+  });
+
+  it('should return null if no cover percentage and facility value', () => {
+    const facilityValueInGBP = null;
+    const coverPercentage = null;
+
+    const result = amendmentHelpers.calculateUkefExposure(facilityValueInGBP, coverPercentage);
+
+    const expected = null;
+
+    expect(result).toEqual(expected);
```

Comment:
if `expected` variable is null, then we can use `expect(result).toBeNull()` instead

Reply:
done